src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDrive.ps1

```diff
@@ -0,0 +1,248 @@
+<#
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License. See License.txt in the project root for license information.
+#>
+
+<#
+.SYNOPSIS
+    Returns a list of all storage drives at a location.
+
+.DESCRIPTION
+    Returns a list of all storage drives at a location.
+
+.PARAMETER Name
+    Name of the storage drive.
+
+.PARAMETER StorageSubSystem
+    Storage subsystem in which the drive is located.
+
+.PARAMETER ScaleUnit
+    Name of the scale unit.
+
+.PARAMETER Location
+    Location of the resource.
+
+.PARAMETER ResourceGroupName
+    Resource group in which the resource provider has been registered.
+
+.PARAMETER ResourceId
+    The resource id.
+
+.PARAMETER Filter
+    OData filter parameter.
+
+.PARAMETER Skip
+    Skip the first N items as specified by the parameter value.
+
+.PARAMETER Top
+    Return the top N items as specified by the parameter value. Applies after the -Skip parameter.
+
+.EXAMPLE
+
+    PS C:\> Get-AzsDrive -ScaleUnit S-Cluster -StorageSubSystem S-Cluster.azurestack.local
+
+    Get a list of all storage drives at a given location.
```

Comment:
Should update example descriptions if you update cmdlet synopsis/description

Reply:
fixed